Gemfile.lock

```diff
@@ -604,12 +603,12 @@ GEM
       tilt (~> 2.0)
     skylight (5.1.1)
       activesupport (>= 5.2.0)
-    sprockets (3.7.2)
+    sprockets (4.0.2)
```

Comment:
Aha, ok, we're trying to move to Sprockets 4 -- https://github.com/rails/sprockets/blob/070fc01947c111d35bb4c836e9bb71962a8e0595/UPGRADING.md#guide-to-upgrading-from-sprockets-3x-to-4x